src/pip/_internal/req/constructors.py

```diff
@@ -258,7 +257,7 @@ def install_req_from_line(
             elif '=' in req and not any(op in req for op in operators):
                 add_msg = "= is not a valid operator. Did you mean == ?"
             else:
-                add_msg = traceback.format_exc()
+                add_msg = ""
```

Comment:
Ideally, I'd say we'd llike an additional `logger.debug()` to have the traceback in verbose mode